tests/onnxruntime/test_modeling.py

```diff
@@ -4146,8 +4169,29 @@ def test_pipeline_text_generation(self, test_name: str, model_arch: str, use_cac
             self.assertEqual(pipe.device, onnx_model.device)
             self.assertIsInstance(outputs[0]["translation_text"], str)
 
+            with tempfile.TemporaryDirectory() as tmpdir:
+                pipe.save_pretrained(tmpdir)
+                model_kwargs = {"use_cache": use_cache}
+                pipe = pipeline(
+                    "translation_en_to_de",
+                    model=tmpdir,
+                    model_kwargs=model_kwargs,
+                    accelerator="ort",
+                )
+                outputs_local_model = pipe(text)
+                self.assertEqual(outputs[0]["translation_text"], outputs_local_model[0]["translation_text"])
+
         gc.collect()
 
+    def test_load_pipeline(self):
+        pipe = pipeline(
+            "text2text-generation",
+            model="echarlaix/t5-small-onnx",
```

Comment:
will add a smaller model in https://huggingface.co/optimum-internal-testing but using this one for now (to do the release asap)